### DISCUSSION TITLE
Result combination Syntax Input

### DISCUSSION THREAD

**Discussion by DannieCW** [Dlubal-Software/RFEM_Python_Client#283] [Q&A] [answered]
Dear developers,

Could you please show me how to input the result combination in Syntax input via API or web services? 
It is much easier this way than specifying per combination item the operators, load type, factors, and brackets. 

Example as below:
![image](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/138588349/8c0d8896-d01a-4cec-84bd-72ba6cbfb824)

Thanks!

Best regards,
Dannie

**Comment by heetrojivadiya** [comment, marked as answer]
Dear @DannieCW,

it sounds interesting to assign LC and CO in result combination as syntax input. Currently, it is automatically derived from list assignment as result combination table and we cannot access syntax input directly. Surely, we are planning to implement accessing syntax input with API or WS. We will let you know, when it will be available. 

Thanks and best regards,
Heet Rojivadiya
